src/ui/views/tui_view.rs

```diff
@@ -4,7 +4,7 @@ use ratatui::widgets::Widget;
 
 use crate::types::option::display::DisplayMode;
 use crate::types::state::AppState;
-use crate::ui::views::{TuiFolderView, TuiHSplitView};
+use crate::ui::views::{tui_minimal_view::TuiMinimalView, TuiFolderView, TuiHSplitView};
```

Comment:
I don't understand why I had to qualify this to make it compile, any help/explanation is appreciated.

Reply:
if you add

```rust
pub use self::tui_minimal_view::*;
```

then it won't require you to do this. Up to you on if you add it though, I don't have a huge preference and will probably change it later anyways xd

Reply:
Thanks for the hint!